sdk/python/tests/integration/registration/test_sql_registry.py

```diff
@@ -112,6 +112,7 @@ def mysql_registry():
     sys.platform == "darwin" and "GITHUB_REF" in os.environ,
     reason="does not run on mac github actions",
 )
+@pytest.mark.integration
```

Comment:
These should be able to run as unit tests - any reason they can't?

Reply:
after discussing, we agreed to make these unit tests; will do in separate PR